tools/pipelines/deploy-website.yml

```diff
@@ -54,6 +55,8 @@ variables:
       value: "$(FLUID_WEBSITE_TORUS_API_TOKEN)"
     ${{ if eq( parameters['deployEnvironment'], 'old') }}:
       value: "$(AZURE_STATIC_WEB_APPS_API_TOKEN)"
+    ${{ if eq(parameters['deployEnvironment'], 'staging') }}:
+      value: "$(STAGING_AZURE_STATIC_WEB_APPS_API_TOKEN)"
```

Comment:
Nit: I think I'd go for `AZURE_STATIC_WEB_APPS_STAGING_API_TOKEN` so they show up next to each other wherever we define/encounter them.